OWML.ModHelper.Input/ModInputCombination.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper.Input
+{
+	public class ModInputCombination : IModInputCombination
+	{
+		private bool _pressed = false, _first = false;
+		private float _firstPressed = 0f, _lastPressed = 0f;
+		private List<KeyCode> _singles = new List<KeyCode>();
+		internal string Combo { get; private set; }
+
+		public ModInputCombination(string combination)
+		{
+			Combo = combination;
+		}
+
+		public float GetLastPressedMoment() { return _lastPressed; }
```

Comment:
Use multiple lines, or this shortcut: 
`public float LastPressedMoment => _lastPressed;`

Reply:
didn't know that way to make getters